frontend/src/utils/constants.ts

```diff
@@ -0,0 +1 @@
+export const MULTI_CONVO_UI_IS_ENABLED = false;
```

Comment:
Why do we have this, I assumed it will be always enabled?

Reply:
This is just the frontend UI, we hope to complete the backend part soon. It is just a matter of merging the two after that and it will be always available